
+ifupdown (0.6.7ubuntu2) dapper; urgency=low
+
+  * Change the dhclient3 leases path to /var/lib/dhcp3 so the leases survive
+    a reboot.  (Ubuntu: #18148)
+
+ -- Scott James Remnant <scott@ubuntu.com>  Wed, 23 Nov 2005 16:11:30 +0000
+

diff -pruN 0.6.7/ifupdown.nw 0.6.7ubuntu7/ifupdown.nw
--- 0.6.7/ifupdown.nw	2005-04-05 18:31:29.000000000 +0100
+++ 0.6.7ubuntu7/ifupdown.nw	2006-01-26 15:56:05.000000000 +0000
@@ -3920,7 +3962,7 @@ method dhcp
 
   up
     [[ifconfig %iface% hw %hwaddress%]]
-    dhclient3 -pf /var/run/dhclient.%iface%.pid -lf /var/run/dhclient.%iface%.leases %iface% \
+    dhclient3 -pf /var/run/dhclient.%iface%.pid -lf /var/lib/dhcp3/dhclient.%iface%.leases %iface% \
         if (execable("/sbin/dhclient3"))
     dhclient -e -pf /var/run/dhclient.%iface%.pid -lf /var/run/dhclient.%iface%.leases %iface% \
         elsif (execable("/sbin/dhclient"))
@@ -3934,7 +3976,7 @@ method dhcp
         elsif (execable("/sbin/dhcpcd"))
 
   down
-    dhclient3 -r -pf /var/run/dhclient.%iface%.pid -lf /var/run/dhclient.%iface%.leases %iface% \
+    dhclient3 -r -pf /var/run/dhclient.%iface%.pid -lf /var/lib/dhcp3/dhclient.%iface%.leases %iface% \
         if (execable("/sbin/dhclient3"))
     cat /var/run/dhclient.%iface%.pid | xargs -i kill -TERM {} \
         elsif (execable("/sbin/dhclient"))
diff -pruN 0.6.7/inet.defn 0.6.7ubuntu7/inet.defn
--- 0.6.7/inet.defn	2005-05-02 14:58:08.000000000 +0100
+++ 0.6.7ubuntu7/inet.defn	2006-01-26 15:59:36.000000000 +0000
@@ -75,7 +75,7 @@ method dhcp
 
   up
     [[ifconfig %iface% hw %hwaddress%]]
-    dhclient3 -pf /var/run/dhclient.%iface%.pid -lf /var/run/dhclient.%iface%.leases %iface% \
+    dhclient3 -pf /var/run/dhclient.%iface%.pid -lf /var/lib/dhcp3/dhclient.%iface%.leases %iface% \
         if (execable("/sbin/dhclient3"))
     dhclient -e -pf /var/run/dhclient.%iface%.pid -lf /var/run/dhclient.%iface%.leases %iface% \
         elsif (execable("/sbin/dhclient"))
@@ -89,7 +89,7 @@ method dhcp
         elsif (execable("/sbin/dhcpcd"))
 
   down
-    dhclient3 -r -pf /var/run/dhclient.%iface%.pid -lf /var/run/dhclient.%iface%.leases %iface% \
+    dhclient3 -r -pf /var/run/dhclient.%iface%.pid -lf /var/lib/dhcp3/dhclient.%iface%.leases %iface% \
         if (execable("/sbin/dhclient3"))
     cat /var/run/dhclient.%iface%.pid | xargs -i kill -TERM {} \
         elsif (execable("/sbin/dhclient"))
